modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -3636,6 +3636,23 @@ channels is derived automatically from src and code.
  */
 CV_EXPORTS_W void cvtColor( InputArray src, OutputArray dst, int code, int dstCn = 0 );
 
+/** @brief Converts an image from dual-mode YUV colorspace to RGB colorspace.
+
+This function only supports 4:2:0 subsampled images as of now.
+
+@param src1: 8-bit image (CV_8U) of the Y plane.
+@param src2: image containing interleaved U/V plane.
+@param dst: output image.
+@param code: Specifies the type of conversion. It can take any of the following values:
+- COLOR_YUV2BGR_NV12
+- COLOR_YUV2RGB_NV12
```

Comment:
Please add `#` before `CV_8U` and `COLOR_` for links generation: https://www.stack.nl/~dimitri/doxygen/manual/autolink.html